<launch>
 <!-- Navigation Program  -->
    <arg name="safe_distance" default="0.2" />
    <arg name="max_distance" default="0.6" />
    <arg name="speed_factor" default="1" />
    <arg name="rotation_speed" default="0.35" />
    <arg name="forward_speed" default="0.1" />
    <arg name="backward_speed" default="-0.1" />
    <arg name="lateral_speed" default="0.1" />
  <node name="wall_follower" pkg="rubot_control" type="rubot_wall_follower_holonomic.py" output="screen" >
    <param name="safe_distance" value="$(arg safe_distance)"/>
    <param name="max_distance" value="$(arg max_distance)"/>
    <param name="speed_factor" value="$(arg speed_factor)"/>
    <param name="rotation_speed" value="$(arg rotation_speed)"/>
    <param name="forward_speed" value="$(arg forward_speed)"/>
    <param name="backward_speed" value="$(arg backward_speed)"/>
    <param name="lateral_speed" value="$(arg lateral_speed)"/>
  </node>
</launch>
